Tests/SentryTests/Helper/SentrySwizzleTests.m

```diff
@@ -353,4 +357,23 @@ - (void)testSwizzleDontCallOriginalImplementation
     XCTAssertThrows([a methodForSwizzlingWithoutCallOriginal]);
 }
 
+- (void)testUnswizzleInstanceMethod
+{
+    SEL methodForUnswizzling = NSSelectorFromString(@"methodForUnswizzling");
+
+    SentrySwizzleTestClass_A *object = [SentrySwizzleTestClass_B new];
+    swizzleVoidMethod(
+        [SentrySwizzleTestClass_A class], methodForUnswizzling, ^{ SentryTestsLog(@"A"); },
+        SentrySwizzleModeAlways, (void *)methodForUnswizzling);
+    [object methodForUnswizzling];
+    ASSERT_LOG_IS(@"A");
+
+    [SentryTestsLog clear];
+
+    [SentrySwizzle unswizzleInstanceMethod:methodForUnswizzling
```

Comment:
What happens if something swizzles after us, and then we call this method? Do we break the call chain? I'd be interested to see a test case for that. 

- some object A has a method `foo`
- swizzle `foo` and replace it with `bar`, which calls through to `foo`
- swizzle `foo` again and replace it with `baz`, which calls through to `bar`
- run unswizzle. does that replace `baz` with `foo`? then whatever was relying on `baz` is broken

Reply:
Good catch!

I will have to further look into that and add handling for cascading swizzling. But as the main point of this PR is isolation of unit tests, I would focus on fixing that.